books/free-programming-books-pt_BR.md

```diff
@@ -193,6 +193,7 @@
 
 ### Java
 
+* [Desenvolvimento de Aplicações Web em Java e Outras Tecnologias](https://github.com/davidbuzatto/Livro-Desenvolvimento-de-Aplica-es-Web-em-Java/releases/download/v1.0ed-1.0c/DesenvolvimentoAplicacoesWebEmJava.-.1ed.-.final.com.correcoes.01.pdf) - David Buzatto
```

Comment:
If you read Contributing guidelines... PDF resources needs mark it respective format

```suggestion
* [Desenvolvimento de Aplicações Web em Java e Outras Tecnologias](https://github.com/davidbuzatto/Livro-Desenvolvimento-de-Aplica-es-Web-em-Java/releases/download/v1.0ed-1.0c/DesenvolvimentoAplicacoesWebEmJava.-.1ed.-.final.com.correcoes.01.pdf) - David Buzatto (PDF)
```

Reply:
I suggest using https://github.com/davidbuzatto/Livro-Desenvolvimento-de-Aplica-es-Web-em-Java/releases so no updates are needed for new versions or added formats

Reply:
I changed the link to the project releases and added the PDF format. Thank you!